sound/soc/sof/intel/hda-ipc.c

```diff
@@ -116,6 +116,12 @@ void hda_dsp_ipc_get_reply(struct snd_sof_dev *sdev)
 	spin_unlock_irqrestore(&sdev->ipc_lock, flags);
 }
 
+static bool hda_dsp_ipc_is_sof(uint32_t msg)
+{
+	return (msg & (HDA_DSP_IPC_PURGE_FW | 0xf << 9)) != msg ||
```

Comment:
0xf << 9? that looks like a magic sequence, can't we make it more readable?

Reply:
According to https://github.com/thesofproject/linux/issues/773#issuecomment-479839611 that's the 4 stream_id bits, that various ROM messages can vary.